apps/studio/package.json

```diff
@@ -22,6 +22,7 @@
     "@graphiql/toolkit": "^0.9.1",
     "@hcaptcha/react-hcaptcha": "^1.8.1",
     "@headlessui/react": "^1.7.17",
+    "@heroicons/react": "^2.1.3",
```

Comment:
Don't tell @kevcodez 👀

An icon wasn't available from lucide? If it's a one-off icon it might be better to just copy and paste the SVG rather than adding a whole package. wdyt?

Reply:
^ this one's from jonny - i think the intention is we'll use icons from both lucide and hero perhaps cc @MildTomato

Reply:
we use both sets

Reply:
it's not a one off, we have a fair few in use that we should start using the lib